index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
```

Comment:
I guess this may be a mistake, since I'm not the one working on the project but I'm still open to help if needed thank you 
@GzimAsani @FabricioLatl